tests/test_variables.py

```diff
@@ -22,6 +22,7 @@ class demo_expression_variable(Variable):
 
 def test_variable_definition():
     """Test variable definition."""
+
```

Comment:
Took me a while to see what you meant. The added blank line? Yes, major improvement! ;-)